scripts/prepare.sh

```diff
@@ -1,13 +1,23 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -eo pipefail
+
+YELLOW='\033[01;33m'
```

Comment:
It would be preferable to use `tput` because it can adapt to varying escape sequence support in the terminals